ci/buildkite-pipeline.sh

```diff
@@ -114,7 +114,7 @@ command_step() {
     timeout_in_minutes: $3
     artifact_paths: "log-*.txt"
     agents:
-      queue: "solana"
+      queue: ${4:-solana}
```

Comment:
is this still valid TOML if we drop the quotes?

Reply:
I think it is yaml 🤔 btw, I found we have quotes and no quotes in the config. I will create a PR to make them all have quotes

<details>
    <summary>steps</summary>

```yaml

steps:

  - name: "sanity"

    command: "ci/test-sanity.sh"

    timeout_in_minutes: 5

    artifact_paths: "log-*.txt"

    agents:

      queue: check

  - wait

  - name: "shellcheck"

    command: "ci/shellcheck.sh"

    timeout_in_minutes: 5

    artifact_paths: "log-*.txt"

    agents:

      queue: check

  - wait

  - name: "checks"

    command: ". ci/rust-version.sh; ci/docker-run.sh $$rust_nightly_docker_image ci/test-checks.sh"

    timeout_in_minutes: 20

    artifact_paths: "log-*.txt"

    agents:

      queue: check

  - wait

  - name: "coverage"

    command: ". ci/rust-version.sh; ci/docker-run.sh $$rust_nightly_docker_image ci/test-coverage.sh"

    timeout_in_minutes: 80

    artifact_paths: "log-*.txt"

    agents:

      queue: solana

  - wait

  - name: "stable"

    command: ". ci/rust-version.sh; ci/docker-run.sh $$rust_stable_docker_image ci/test-stable.sh"

    timeout_in_minutes: 70

    artifact_paths: "log-*.txt"

    agents:

      queue: solana

  - name: "doctest"

    command: "ci/test-docs.sh"

    timeout_in_minutes: 15

    artifact_paths: "log-*.txt"

    agents:

      queue: solana

  - wait

  - command: "ci/test-stable-sbf.sh"

    name: "stable-sbf"

    timeout_in_minutes: 35

    artifact_paths: "sbf-dumps.tar.bz2"

    agents:

      queue: "solana"

  - command: "ci/test-stable-perf.sh"

    name: "stable-perf"

    timeout_in_minutes: 35

    artifact_paths: "log-*.txt"

    agents:

      queue: "cuda"

  - command: "scripts/build-downstream-projects.sh"

    name: "downstream-projects"

    timeout_in_minutes: 35

    agents:

      queue: "solana"

  - name: "wasm"

    command: ". ci/rust-version.sh; ci/docker-run.sh $$rust_stable_docker_image ci/test-wasm.sh"

    timeout_in_minutes: 20

    artifact_paths: "log-*.txt"

    agents:

      queue: solana

  - name: "bench"

    command: "ci/test-bench.sh"

    timeout_in_minutes: 40

    artifact_paths: "log-*.txt"

    agents:

      queue: solana

  - name: "local-cluster"

    command: ". ci/rust-version.sh; ci/docker-run.sh $$rust_stable_docker_image ci/test-local-cluster.sh"

    timeout_in_minutes: 40

    artifact_paths: "log-*.txt"

    agents:

      queue: solana

  - name: "local-cluster-flakey"

    command: ". ci/rust-version.sh; ci/docker-run.sh $$rust_stable_docker_image ci/test-local-cluster-flakey.sh"

    timeout_in_minutes: 10

    artifact_paths: "log-*.txt"

    agents:

      queue: solana

  - name: "local-cluster-slow-1"

    command: ". ci/rust-version.sh; ci/docker-run.sh $$rust_stable_docker_image ci/test-local-cluster-slow-1.sh"

    timeout_in_minutes: 40

    artifact_paths: "log-*.txt"

    agents:

      queue: solana

  - name: "local-cluster-slow-2"

    command: ". ci/rust-version.sh; ci/docker-run.sh $$rust_stable_docker_image ci/test-local-cluster-slow-2.sh"

    timeout_in_minutes: 40

    artifact_paths: "log-*.txt"

    agents:

      queue: solana
```

</details>

Reply:
we don't need the PR anymore cuz the no quote string is also from here. when I added quote, the problem is solved 🤞

Reply:
nvm. I think I need more coffee to reboot myself. 😴 

I added the quotes for not changing the behavior

Reply:
thanks for investigating!